src/common/utils/gpo.h

```diff
@@ -115,6 +118,128 @@ namespace powertoys_gpo {
         }
     }
 
+    inline std::string getPolicyListValue(const std::wstring& registry_list_enabled_value, const std::wstring& registry_list_path, const std::wstring& registry_list_value_name)
```

Comment:
@jaimecbernardo
Is it possible for you to look at this function. It returns wrong strings when reading the REG_SZ values. It would be amazing if you can have some time on helping me.

Reply:
I have found my bug and fixed it.